service/orchestrator/metrics.go

```diff
@@ -60,12 +60,13 @@ func (*Service) CreateMetric(_ context.Context, req *orchestrator.CreateMetricRe
 	// Validate the metric request
 	err = req.Metric.Validate(assessment.WithMetricRequiresId())
 	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "Validation of metric failed: %v", err)
+		log.Errorf("Validation of metric failed: %v", err)
+		return nil, status.Error(codes.InvalidArgument, "validation of metric failed")
```

Comment:
Why removing the error message which contains the specific part of the validation error?

Reply:
Here we could also avoid writing same string twice